src/main/java/io/sniffy/servlet/BufferedPrintWriter.java

```diff
@@ -0,0 +1,13 @@
+package io.sniffy.servlet;
+
+import java.io.PrintWriter;
+
+public class BufferedPrintWriter extends PrintWriter {
+
+    private final BufferedServletOutputStream bufferedServletOutputStream;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'bufferedServletOutputStream' could be replaced by a local variable.](https://www.codacy.com/app/sniffy/sniffy/file/2184256033/issues/source?bid=3005960&fileBranchId=3165140#l7)